### ISSUE TITLE
Configure stack frames or disable warnings about it

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/briefcase#400]
#### Software versions 
Briefcase v1.x.x, Java v1.x.x, operating system, Aggregate v1.x.x, Collect v1.x.x...

#### Problem description
When running the logback.xml release config with `./gradlew run`, I get this message

```
2018-02-26 10:22:11,226 [main] WARN  io.sentry.DefaultSentryClientFactory - No 'stacktrace.app.packages' was configured, this option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation: https://docs.sentry.io/clients/java/config/#in-application-stack-frames
```

#### Expected behavior
Perhaps we should configure stack frames? https://docs.sentry.io/clients/java/config/#in-application-stack-frames

**Comment by ggalmazor** [comment]
After #391, this is no longer happening :P

**Comment by yanokwa** [comment]
Fixed by #391
